projects/demo/src/modules/directives/dropdown/index.html

```diff
@@ -78,24 +78,27 @@
                 </ng-template>
             </span>
         </tui-doc-demo>
-        <tui-doc-documentation>
-            <ng-template
-                documentationPropertyMode="input"
-                documentationPropertyName="tuiDropdownManual"
-                documentationPropertyType="boolean"
-                [(documentationPropertyValue)]="open"
+        <table tuiDocAPI>
+            <tr
+                name="[tuiDropdownManual]"
+                tuiDocAPIItem
+                type="boolean"
+                [(value)]="open"
             >
                 Show dropdown (basic manual implementation, see related pages for other options)
-            </ng-template>
-            <ng-template
-                documentationPropertyMode="input"
-                documentationPropertyName="tuiDropdown"
-                documentationPropertyType="PolymorpheusContent"
+            </tr>
+            <tr
+                name="[tuiDropdown]"
+                tuiDocAPIItem
+                type="PolymorpheusContent"
             >
                 Content
-            </ng-template>
-            <dropdown-documentation />
-        </tui-doc-documentation>
+            </tr>
+            <tbody
+                #dropdown
+                tuiDocDropdown
```

Comment:
Looks like we need to be able to show/hide certain options. For example, here we do not need `[(open)]` and in appearance we also need to able to hide appearance, because it is not changeable for checkbox/radio/switch, although other appearance options are applicable. In appearance let's hide it if we passed empty array.

Reply:
Added `hiddenOptions`